src/main/java/de/embl/cba/mobie/n5/zarr/OMEZarrReader.java

```diff
@@ -38,7 +40,7 @@ private SpimData readFile() throws IOException
         N5OMEZarrImageLoader.logChunkLoading = logChunkLoading;
         N5OmeZarrReader reader = new N5OmeZarrReader(this.filePath, new GsonBuilder());
         HashMap<String, Integer> axesMap = reader.getAxes();
-        N5OMEZarrImageLoader imageLoader = new N5OMEZarrImageLoader(reader, axesMap);
+        N5OMEZarrImageLoader imageLoader = new N5OMEZarrImageLoader(reader, axesMap, MoBIE.sharedQueue);
```

Comment:
@KateMoreva here the global shared queue is injected into the ImageLoader. We should enable this for all ImageLoaders, not only the OMEZarr ones.

Reply:
Good point, I guess the functions in bdv-imageloader could have a way to accept a SharedQueue from whoever calls them. Or (less nice) there could be some global flag within bdv-imageloader to use a SharedQueue. I'd leave it up to you to come up with some good idea 😄  👍

Reply:
Ok. There will be no problems with accepting SharedQueue in loaders (they are called from MoBIE).